core/tests/protractor_utils/waitFor.js

```diff
@@ -0,0 +1,104 @@
+// Copyright 2018 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Utilities for delaying actions with Protractor's
+ * ExpectedConditions.
+ */
+
+var until = protractor.ExpectedConditions;
+var waitTime = 5000;
+
+/**
+ * @param {string} errorMessage - Error message when alert does not pop up.
+ * @param {int} [waitTime] - Optional wait time (in milliseconds) parameter.
+ */
+var alertIsPresent = function(errorMessage) {
```

Comment:
alertToBePresent perhaps (so that it can be read as waitFor.alertToBePresent)

Or, alertToAppear (or alertDialogToAppear)

Reply:
Going with alertToBePresent!